### DISCUSSION TITLE
Plotly code generation

### DISCUSSION THREAD

**Discussion by thi-cros** [vanna-ai/vanna#662] [General]
Hi,

I am experimenting the method **generate_plotly_code()**. I saw that there is a parameter called **chart_instructions** defined as follows:
_chart_instructions_ (str): Optional instructions for how to plot the chart.
I am trying to tell him to prefer bar plot over scatter plot but it doesn't work.
Does someone ever try it ? Is it still maintained ?

Thanks for your help !

**Comment by a29874750a** [comment]
Hi,
I think you can modify the prompt in generate_plotly_code() directly.
